### DISCUSSION TITLE
Bitbucket.org RSA key changed

### DISCUSSION THREAD

**Discussion by cbid71** [argoproj/argo-helm#2128] [General]
Hello guys,

Just a small message to tell you that the bitbucket.org rsa key referenced in configs.ssh.knownhosts in the chart values.yaml is now invalid.

https://bitbucket.org/blog/ssh-host-key-changes

I'm new on the place and I get 403 when I try to push my working branch to create a PR.

If somebody want to manage the modification, have fun, the new RSA key for bitbucket.org is on the link.

Have a good day !

**Comment by yu-croco** [comment]
Hi @cbid71 , this is talked at https://github.com/argoproj/argo-helm/issues/2059 and it has been resolved on argo-cd v5.34.1.

**Reply by cbid71** [reply]
Oh, thank you for the information !
